### ISSUE TITLE
New tagged version/release for device plugins v1beta1

### ISSUE THREAD

**Issue by mlsorensen** [kubevirt/device-plugin-manager#8]
It would be nice if the version in master that references k8s.io/kubernetes/pkg/kubelet/apis/deviceplugin/v1beta1 could be tagged. My dependency manager pulls v1.9.2 by default, which is using device plugins v1alpha1.

**Comment by phoracek** [comment]
@mlsorensen sure, there you go https://github.com/kubevirt/device-plugin-manager/releases/tag/v1.10.0. Please let me know if it is good enough or you need a separate branch too.

**Comment by mlsorensen** [comment]
Works great, thanks.  Is there any reason why kubevirt's virt-handler doesn't use this for /dev/kvm? Just looking to understand if this code base is deprecated or if there were other extenuating circumstances.

**Comment by phoracek** [comment]
@mlsorensen we created this framework a while ago to support our device plugin efforts, one of those was /dev/kvm, but we also played with VFIO passthrough and wanted to use it for networking.

Now for /dev/kvm, I believe the intention was (but it was long ago and my memory is weak) to keep kubevirt/kubevirt slimmer, without external dependencies. The /dev/kvm DP is pretty simple after all and mandatory for core kubevirt.

For networking, since the time we were focused on it, Multus (https://github.com/intel/multus-cni) became the de-facto standard for us and the way to go.

While those mentioned DPs are not used anymore, this framework should be fine and I'd encourage you to give it a try with you DPs if it makes your development easier. There are no flaws I'm aware of. If you find any, I'd be happy to assist.
